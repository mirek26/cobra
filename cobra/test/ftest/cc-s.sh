$COBRA_HOME/cobra cc.py > /dev/null;
echo "4 2 3 0 1 0" | $COBRA_HOME/cobra-backend -m s .cobra.in
echo "0 2 2 2 2 2 2 2 2 2 2 0 2 0" | $COBRA_HOME/cobra-backend -m s .cobra.in
