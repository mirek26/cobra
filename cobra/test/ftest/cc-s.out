
[1;97m===== SIMULATION =====[0m
Select an experiment: 
0) weighing1 [ 1 2 ]  - M: 2 2 20 F: 10 10 2 
1) weighing2 [ 1 2 3 4 ]  - M: 4 4 16 F: 8 8 4 
2) weighing3 [ 1 2 3 4 5 6 ]  - M: 6 6 12 F: 6 6 6 
3) weighing4 [ 1 2 3 4 5 6 7 8 ]  - M: 8 8 8 F: 4 4 8 
4) weighing5 [ 1 2 3 4 5 6 7 8 9 10 ]  - M: 10 10 4 F: 2 2 10 
5) weighing6 [ 1 2 3 4 5 6 7 8 9 10 11 12 ]  - M: 12 12 0 F: 0 0 26 
> [32mEXPERIMENT: weighing5 1 2 3 4 5 6 7 8 9 10 [0m
Select an outcome: 
0) lighter 
1) heavier 
2) same 
> [32mOUTCOME: same
  ->   Â¬(x1 âˆ¨ x2 âˆ¨ x3 âˆ¨ x4 âˆ¨ x5 âˆ¨ x6 âˆ¨ x7 âˆ¨ x8 âˆ¨ x9 âˆ¨ x10)

[0mSelect an experiment: 
1) weighing1 [ 1 11 ]  - M: 1 1 2 F: 13 13 12 
2) weighing1 [ 11 12 ]  - M: 2 2 0 F: 10 10 26 
3) weighing2 [ 1 2 11 12 ]  - M: 2 2 0 F: 11 11 26 
> [32mEXPERIMENT: weighing2 1 2 11 12 [0m
Select an outcome: 
0) lighter 
1) heavier 
-) same (unsatisfiable)
> [32mOUTCOME: lighter
  ->   (((x1 âˆ¨ x2) âˆ§ Â¬y) âˆ¨ ((x11 âˆ¨ x12) âˆ§ y))

[0mSelect an experiment: 
1) weighing1 [ 1 11 ]  - M: 1 0 1 F: 13 26 13 
2) weighing1 [ 11 12 ]  - M: 1 1 0 F: 13 13 26 
> [32mEXPERIMENT: weighing1 1 11 [0m
Select an outcome: 
0) lighter 
-) heavier (unsatisfiable)
2) same 
> [32mOUTCOME: lighter
  ->   ((x1 âˆ§ Â¬y) âˆ¨ (x11 âˆ§ y))

[0m[1;97mSOLVED in 3 experiments![0m
TRUE: x11 y 
FALSE: x1 x2 x3 x4 x5 x6 x7 x8 x9 x10 x12 


[1;97m===== SIMULATION =====[0m
Select an experiment: 
0) weighing1 [ 1 2 ]  - M: 2 2 20 F: 10 10 2 
1) weighing2 [ 1 2 3 4 ]  - M: 4 4 16 F: 8 8 4 
2) weighing3 [ 1 2 3 4 5 6 ]  - M: 6 6 12 F: 6 6 6 
3) weighing4 [ 1 2 3 4 5 6 7 8 ]  - M: 8 8 8 F: 4 4 8 
4) weighing5 [ 1 2 3 4 5 6 7 8 9 10 ]  - M: 10 10 4 F: 2 2 10 
5) weighing6 [ 1 2 3 4 5 6 7 8 9 10 11 12 ]  - M: 12 12 0 F: 0 0 26 
> [32mEXPERIMENT: weighing1 1 2 [0m
Select an outcome: 
0) lighter 
1) heavier 
2) same 
> [32mOUTCOME: same
  ->   Â¬(x1 âˆ¨ x2)

[0mSelect an experiment: 
1) weighing1 [ 1 3 ]  - M: 1 1 18 F: 13 13 3 
2) weighing1 [ 3 4 ]  - M: 2 2 16 F: 10 10 4 
3) weighing2 [ 1 2 3 4 ]  - M: 2 2 16 F: 11 11 4 
4) weighing2 [ 1 3 4 5 ]  - M: 3 3 14 F: 9 9 5 
5) weighing2 [ 3 4 5 6 ]  - M: 4 4 12 F: 8 8 6 
6) weighing3 [ 1 2 3 4 5 6 ]  - M: 4 4 12 F: 8 8 6 
7) weighing3 [ 1 3 4 5 6 7 ]  - M: 5 5 10 F: 7 7 7 
8) weighing3 [ 3 4 5 6 7 8 ]  - M: 6 6 8 F: 6 6 8 
9) weighing4 [ 1 2 3 4 5 6 7 8 ]  - M: 6 6 8 F: 6 6 8 
10) weighing4 [ 1 3 4 5 6 7 8 9 ]  - M: 7 7 6 F: 5 5 9 
11) weighing4 [ 3 4 5 6 7 8 9 10 ]  - M: 8 8 4 F: 4 4 10 
12) weighing5 [ 1 2 3 4 5 6 7 8 9 10 ]  - M: 8 8 4 F: 4 4 10 
13) weighing5 [ 1 3 4 5 6 7 8 9 10 11 ]  - M: 9 9 2 F: 3 3 12 
14) weighing5 [ 3 4 5 6 7 8 9 10 11 12 ]  - M: 10 10 0 F: 2 2 26 
15) weighing6 [ 1 2 3 4 5 6 7 8 9 10 11 12 ]  - M: 10 10 0 F: 2 2 26 
> [32mEXPERIMENT: weighing1 3 4 [0m
Select an outcome: 
0) lighter 
1) heavier 
2) same 
> [32mOUTCOME: same
  ->   Â¬(x3 âˆ¨ x4)

[0mSelect an experiment: 
1) weighing1 [ 1 5 ]  - M: 1 1 14 F: 13 13 5 
2) weighing1 [ 5 6 ]  - M: 2 2 12 F: 10 10 6 
3) weighing2 [ 1 2 5 6 ]  - M: 2 2 12 F: 11 11 6 
4) weighing2 [ 1 5 6 7 ]  - M: 3 3 10 F: 9 9 7 
5) weighing2 [ 5 6 7 8 ]  - M: 4 4 8 F: 8 8 8 
6) weighing3 [ 1 2 3 5 6 7 ]  - M: 3 3 10 F: 10 10 7 
7) weighing3 [ 1 2 5 6 7 8 ]  - M: 4 4 8 F: 8 8 8 
8) weighing3 [ 1 5 6 7 8 9 ]  - M: 5 5 6 F: 7 7 9 
9) weighing3 [ 5 6 7 8 9 10 ]  - M: 6 6 4 F: 6 6 10 
10) weighing4 [ 1 2 3 4 5 6 7 8 ]  - M: 4 4 8 F: 9 9 8 
11) weighing4 [ 1 2 3 5 6 7 8 9 ]  - M: 5 5 6 F: 7 7 9 
12) weighing4 [ 1 2 5 6 7 8 9 10 ]  - M: 6 6 4 F: 6 6 10 
13) weighing4 [ 1 5 6 7 8 9 10 11 ]  - M: 7 7 2 F: 5 5 12 
14) weighing4 [ 5 6 7 8 9 10 11 12 ]  - M: 8 8 0 F: 4 4 26 
15) weighing5 [ 1 2 3 4 5 6 7 8 9 10 ]  - M: 6 6 4 F: 6 6 10 
16) weighing5 [ 1 2 3 5 6 7 8 9 10 11 ]  - M: 7 7 2 F: 5 5 12 
17) weighing5 [ 1 2 5 6 7 8 9 10 11 12 ]  - M: 8 8 0 F: 4 4 26 
18) weighing6 [ 1 2 3 4 5 6 7 8 9 10 11 12 ]  - M: 8 8 0 F: 4 4 26 
> [32mEXPERIMENT: weighing1 5 6 [0m
Select an outcome: 
0) lighter 
1) heavier 
2) same 
> [32mOUTCOME: same
  ->   Â¬(x5 âˆ¨ x6)

[0mSelect an experiment: 
1) weighing1 [ 1 7 ]  - M: 1 1 10 F: 13 13 7 
2) weighing1 [ 7 8 ]  - M: 2 2 8 F: 10 10 8 
3) weighing2 [ 1 2 7 8 ]  - M: 2 2 8 F: 11 11 8 
4) weighing2 [ 1 7 8 9 ]  - M: 3 3 6 F: 9 9 9 
5) weighing2 [ 7 8 9 10 ]  - M: 4 4 4 F: 8 8 10 
6) weighing3 [ 1 2 3 7 8 9 ]  - M: 3 3 6 F: 10 10 9 
7) weighing3 [ 1 2 7 8 9 10 ]  - M: 4 4 4 F: 8 8 10 
8) weighing3 [ 1 7 8 9 10 11 ]  - M: 5 5 2 F: 7 7 12 
9) weighing3 [ 7 8 9 10 11 12 ]  - M: 6 6 0 F: 6 6 26 
10) weighing4 [ 1 2 3 4 7 8 9 10 ]  - M: 4 4 4 F: 9 9 10 
11) weighing4 [ 1 2 3 7 8 9 10 11 ]  - M: 5 5 2 F: 7 7 12 
12) weighing4 [ 1 2 7 8 9 10 11 12 ]  - M: 6 6 0 F: 6 6 26 
13) weighing5 [ 1 2 3 4 5 7 8 9 10 11 ]  - M: 5 5 2 F: 8 8 12 
14) weighing5 [ 1 2 3 4 7 8 9 10 11 12 ]  - M: 6 6 0 F: 6 6 26 
15) weighing6 [ 1 2 3 4 5 6 7 8 9 10 11 12 ]  - M: 6 6 0 F: 7 7 26 
> [32mEXPERIMENT: weighing1 7 8 [0m
Select an outcome: 
0) lighter 
1) heavier 
2) same 
> [32mOUTCOME: same
  ->   Â¬(x7 âˆ¨ x8)

[0mSelect an experiment: 
1) weighing1 [ 1 9 ]  - M: 1 1 6 F: 13 13 9 
2) weighing1 [ 9 10 ]  - M: 2 2 4 F: 10 10 10 
3) weighing2 [ 1 2 9 10 ]  - M: 2 2 4 F: 11 11 10 
4) weighing2 [ 1 9 10 11 ]  - M: 3 3 2 F: 9 9 12 
5) weighing2 [ 9 10 11 12 ]  - M: 4 4 0 F: 8 8 26 
6) weighing3 [ 1 2 3 9 10 11 ]  - M: 3 3 2 F: 10 10 12 
7) weighing3 [ 1 2 9 10 11 12 ]  - M: 4 4 0 F: 8 8 26 
8) weighing4 [ 1 2 3 4 9 10 11 12 ]  - M: 4 4 0 F: 9 9 26 
> [32mEXPERIMENT: weighing1 9 10 [0m
Select an outcome: 
0) lighter 
1) heavier 
2) same 
> [32mOUTCOME: same
  ->   Â¬(x9 âˆ¨ x10)

[0mSelect an experiment: 
1) weighing1 [ 1 11 ]  - M: 1 1 2 F: 13 13 12 
2) weighing1 [ 11 12 ]  - M: 2 2 0 F: 10 10 26 
3) weighing2 [ 1 2 11 12 ]  - M: 2 2 0 F: 11 11 26 
> [32mEXPERIMENT: weighing1 11 12 [0m
Select an outcome: 
0) lighter 
1) heavier 
-) same (unsatisfiable)
> [32mOUTCOME: lighter
  ->   ((x11 âˆ§ Â¬y) âˆ¨ (x12 âˆ§ y))

[0mSelect an experiment: 
1) weighing1 [ 1 11 ]  - M: 0 1 1 F: 26 13 13 
2) weighing1 [ 1 12 ]  - M: 1 0 1 F: 13 26 13 
4) weighing2 [ 1 2 11 12 ]  - M: 1 1 0 F: 13 13 26 
> [32mEXPERIMENT: weighing1 1 12 [0m
Select an outcome: 
0) lighter 
-) heavier (unsatisfiable)
2) same 
> [32mOUTCOME: lighter
  ->   ((x1 âˆ§ Â¬y) âˆ¨ (x12 âˆ§ y))

[0m[1;97mSOLVED in 7 experiments![0m
TRUE: x12 y 
FALSE: x1 x2 x3 x4 x5 x6 x7 x8 x9 x10 x11 

